.vitepress/theme/components/playground/InlinePlayground.vue

```diff
@@ -316,3 +316,13 @@ onMounted(async() => {
   }
 }
 </style>
+
+<style scoped>
+@media (min-width: 1600px) {
+  .inline-playground {
+    @apply mt-8 mb-8;
+    width: 150%;
+    margin-left: -25%;
+  }
+}
+</style>
```

Comment:
This will break mobile view, and is not only used in homepage.

Reply:
this was only for desktop `@media (min-width: 1600px)`, but I'll revert this change as it can have unexpected consequences if not within the content pages